voyager-publish/test/test.js

```diff
@@ -24,9 +24,6 @@ describe('unit', () => {
       release () {},
       async query (statement, params) {
         clientStatements.push(statement)
-        if (statement.includes('SELECT COUNT(*) FROM measurements')) {
-          return { rows: [{ count: 10 }] }
-        }
```

Comment:
My change did not break the tests, making me believe this block is unnecessary (not used by the tests).

Reply:
Weird. The tests are failing in CI.

```  1) unit
       publishes:
     TypeError: Cannot read properties of undefined (reading 'count')
      at publish (file:///home/runner/work/voyager-api/voyager-api/voyager-publish/index.js:36:14)
      at async Context.<anonymous> (file:///home/runner/work/voyager-api/voyager-api/voyager-publish/test/test.js:73:5)
```

Reply:
see https://github.com/filecoin-station/voyager-api/pull/28#discussion_r1590887825

Reply:
Fixed in [6458066](https://github.com/filecoin-station/voyager-api/pull/28/commits/6458066b0d0862a395a590064c3899d998cadfd9) (hopefully)